vast.yaml.example

```diff
@@ -102,7 +102,7 @@ vast:
     # Configures if and how metrics should be ingested back into VAST.
     self-sink:
       enable: true
-      slice-size: 100
+      slice-size: 128
```

Comment:
This change is not documented in the changelog. Is it not important?

Reply:
I'd argue it is not.